rasa_nlu/test.py

```diff
@@ -613,42 +600,42 @@ def extract_confidence(result):  # pragma: no cover
     return result.get('intent', {}).get('confidence')
 
 
-def get_predictions(interpreter,
-                    test_data,
-                    intent_targets):  # pragma: no cover
-    """Run the model for the test set and extracts intents and entities.
+def get_eval_data(interpreter, test_data):  # pragma: no cover
+    """Runs the model for the test set and extracts targets and predictions
+    for both entities and intents.
 
-    Return intent and entity predictions, the original messages and the
-    confidences of the predictions."""
+    Returns intent results (intent targets and predictions, the original
+    messages and the confidences of the predictions), as well as entity
+    targets, predictions, and tokens."""
 
     logger.info("Running model for predictions:")
 
-    intent_results, entity_predictions, tokens = [], [], []
+    intent_results, entity_targets, entity_predictions, tokens = [], [], [], []
```

Comment:
i'd definitely like to clean it up (check out "things we could do" in the main description) -- I was looking for input into that, whether we should make named tuples for entity results or just put it all into an evaluation store (which would look a little different from the core one, since the core includes actions and this would include tokens)

Reply:
oh sorry i assumed when you requested a review these were the final changes :D

Reply:
i mean this is the question, not sure when we're merging the Core and NLU test scripts/if we even are -- @tmbo ?

Reply:
there is no plan to do that at the moment - but we consider that if it makes sense. but I think there are different things to evaluate (e.g. entities vs action predictions) that we do not necessarily need to mix code wise. from a users perspective running the CLI scripts it might still make sense to compare them.